### ISSUE TITLE
Huge accuracy decrease after upgrading to ImageAI 2.1.6 and TensorFlow 2.4.0

### ISSUE THREAD

**Issue by Zordiac** [OlafenwaMoses/ImageAI#669]
Hi @OlafenwaMoses 

I recently upgraded to ImageAI 2.1.6, TensorFlow 2.4.0 and Python 3.7. I then retrained my model with the exact same dataset and same configuration, and the loss was about 5 times as high as before. After testing, the model performed really really bad on my evaluation data set.

I downgraded to ImageAI 2.1.5, TensorFlow 1.12 and Python 3.6 to verify my results. I trained my model again with the same dataset, and the models loss was very good and the detections were fine again.

What is wrong with the latest ImageAI version? I used exactly the same dataset, and the results are so different. The latest version does not work at all.

Thanks in advance.

**Comment by DayTimeDreamer** [comment]
Hi, I have the same problem to you. Can you tell me the dependences' version of ImageAI 2.1.5. 

Thanks in advance.

**Comment by Zordiac** [comment]
Sure.

For ImageAI 2.1.5:
Python 3.6
TensorFlow_GPU 1.15.3
Keras 2.3.1

**Comment by DayTimeDreamer** [comment]
Thank you！
I've used TensorFlow 1.12.3.  But the training is slower than TensorFlow 2.4.0.
